src/nginxconfig/util/backwards_compatibility.js

```diff
@@ -36,6 +36,8 @@ const globalMap = {
     directory_letsencrypt:      ['https', 'letsEncryptRoot'],
 
     referrer_policy:            ['security', 'referrerPolicy'],
+    security_txt:               ['security', 'securityTxt', oldBool],
+    security_txt_path:          ['security', 'securityTxtPreferredLanguages'],
```

Comment:
```suggestion
```
These aren't needed, settings for security.txt never existed on the old version of the tool, so there is no backwards compatibility to support.